packages/eslint-plugin-lit-a11y/tests/lib/rules/click-events-have-key-events.js

```diff
@@ -40,93 +40,55 @@ ruleTester.run('click-events-have-key-events', rule, {
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<main @click=${foo}></main>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
     },
     {
-      code: 'html`<article @click=${foo} ></article>`;',
-      errors: [
-        {
-          message:
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<article @click=${foo}></article>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
     },
     {
-      code: 'html`<header @click=${foo} ></header>`;',
-      errors: [
-        {
-          message:
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<header @click=${foo}></header>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
     },
     {
-      code: 'html`<footer @click=${foo} ></footer>`;',
-      errors: [
-        {
-          message:
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<footer @click=${foo}></footer>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
     },
     {
-      code: 'html`<a @click=${foo} ></a>`;',
-      errors: [
-        {
-          message:
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<a @click=${foo}></a>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
     },
     {
-      code: 'html`<custom-button @click=${foo} ></custom-button>`;',
-      errors: [
-        {
-          message:
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<custom-button @click=${foo}></custom-button>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
+      options: [{ allowCustomElements: false }],
     },
     {
-      code: 'html`<another-button @click=${foo} ></another-button>`;',
-      errors: [
-        {
-          message:
-            'Clickable non-interactive elements must have at least 1 keyboard event listener',
-        },
-      ],
+      code: 'html`<another-button @click=${foo}></another-button>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
+      options: [{ allowCustomElements: false }],
+    },
+    {
+      code: 'html`<custom-button @click=${foo}></custom-button>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
+      options: [{ allowCustomElements: false, allowList: ['another-button'] }],
+    },
+    {
+      code: 'html`<another-button @click=${foo}></another-button>`;',
+      errors: [{ messageId: 'clickableNonInteractiveElements' }],
+      options: [{ allowCustomElements: false, allowList: ['custom-button'] }],
```

Comment:
Do we have a `valid` test with `allowCustomElements: true`?

Reply:
it's the default, but i'll explicitly add some test cases

Reply:
side note: semantically, `allowList` only makes sense when `allowCustomElements: false`, but I'm not sure if JSONSchema provides for that kind of validation

Reply:
> it's the default, but i'll explicitly add some test cases

ah yes, in that case its fair enough 👍